docs/api/_data/repository-signatures-index.json

```diff
@@ -0,0 +1,15 @@
+{
+  "allRepositorySigned": false,
+  "signingCertificates": [
+    {
+      "fingerprints": {
+        "2.16.840.1.101.3.4.2.1": "79851da778a4780d2720588901a41b70ad9e584f847d09359949862f138e92ac"
```

Comment:
This test data will be updated once we know the real certificate. This is not critical but makes a better example.

Reply:
This is done now.